components/CollectiveNavbar.js

```diff
@@ -236,7 +367,167 @@ const CollectiveNavbar = ({
+            </Container>
+          </LinkCollective>
+        </AvatarBox>
+        <Box display={['block', null, null, 'none']}>
+          <CollectiveNameV2
+            mx={2}
+            py={2}
+            fontSize={['16px', '20px']}
+            lineHeight={['24px', '28px']}
+            textAlign="center"
+            fontWeight="500"
+            color="black.800"
+          >
+            {isLoading ? (
+              <LoadingPlaceholder height={14} minWidth={100} />
+            ) : (
+              <LinkCollective collective={collective} onClick={onCollectiveClick} />
+            )}
+          </CollectiveNameV2>
+        </Box>
+        {!onlyInfos && (
+          <Box display={['block', 'none']} marginLeft="auto">
+            <ExpandMenuIcon onClick={() => setExpanded(!isExpanded)} />
+          </Box>
+        )}
+      </InfosContainerV2>
+      {/** Main navbar items */}
+      {/* TO DO: they don't shrink properly when the container gets small - needs work */}
+      <Container
+        backgroundColor="#fff"
+        zIndex={1}
+        display={isExpanded ? 'flex' : ['none', 'flex']}
+        flexDirection={['column', 'row']}
+        flexBasis="fit-content"
+        flexShrink={2}
+        flexGrow={1}
+        justifyContent={['space-between', 'flex-start']}
+        minWidth={0}
+        order={[0, 3, 0]}
+        borderTop={['none', '1px solid #e1e1e1', 'none']}
+      >
+        {isLoading ? (
+          <LoadingPlaceholder height={43} minWidth={150} mb={2} />
+        ) : (
+          <Fragment>
+            {sections.map(section => (
+              <MenuLinkContainerV2
+                mr={[0, 3]}
+                key={section}
+                isSelected={section === selected}
+                onClick={() => {
+                  if (isExpanded) {
+                    setExpanded(false);
+                  }
+                  if (onSectionClick) {
+                    onSectionClick(section);
+                  }
+                }}
+              >
+                <Flex py={2} px={[3, 0]}>
+                  <Flex alignItems="center" mr={2}>
+                    <IconIllustration src={getCollectiveNavbarIcon(section)} />
+                  </Flex>
+                  <Flex alignItems="center">
+                    <MenuLinkV2
+                      as={LinkComponent}
+                      collectivePath={collective.path || `/${collective.slug}`}
+                      section={section}
+                      label={i18nCollectivePageSection(intl, section)}
+                    />
+                  </Flex>
+                </Flex>
+              </MenuLinkContainerV2>
+            ))}
+          </Fragment>
+        )}
+      </Container>
+      {/* CTAs for v2 navbar & admin panel */}
+      <Container
+        display={isExpanded ? 'flex' : ['none', 'flex']}
+        flexDirection={['column', 'row']}
+        flexBasis="fit-content"
+        marginLeft={['none', 'auto']}
```

Comment:
`margin: none` is not valid CSS. You probably meant:

```suggestion
        marginLeft={0, 'auto']}
```